src/main/java/io/kestra/plugin/spark/SparkCLI.java

```diff
@@ -38,43 +38,37 @@
-                    tasks:
-                    - id: local-file
-                      type: io.kestra.plugin.core.storage.LocalFiles
-                      inputs:
-                        pi.py: |
-                          import sys
-                          from random import random
-                          from operator import add
-                          from pyspark.sql import SparkSession
+                  - id: hello
+                    type: io.kestra.plugin.spark.SparkCLI
+                    inputFiles:
+                      pi.py: |
+                        import sys
+                        from random import random
+                        from operator import add
+                        from pyspark.sql import SparkSession
                 
+                        if __name__ == "__main__":
+                            spark = SparkSession \
+                                .builder \
+                                .appName("PythonPi") \
+                                .getOrCreate()
                 
-                          if __name__ == "__main__":
-                              spark = SparkSession \\
-                                  .builder \\
-                                  .appName("PythonPi") \\
-                                  .getOrCreate()
+                            partitions = int(sys.argv[1]) if len(sys.argv) > 1 else 2
+                            n = 100000 * partitions
                 
-                              partitions = int(sys.argv[1]) if len(sys.argv) > 1 else 2
-                              n = 100000 * partitions
+                            def f(_: int) -> float:
+                                x = random() * 2 - 1
+                                y = random() * 2 - 1
+                                return 1 if x ** 2 + y ** 2 <= 1 else 0
                 
-                              def f(_: int) -> float:
-                                  x = random() * 2 - 1
-                                  y = random() * 2 - 1
-                                  return 1 if x ** 2 + y ** 2 <= 1 else 0
+                            count = spark.sparkContext.parallelize(range(1, n + 1), partitions).map(f).reduce(add)
+                            print("Pi is roughly %f" % (4.0 * count / n))
                 
-                              count = spark.sparkContext.parallelize(range(1, n + 1), partitions).map(f).reduce(add)
-                              print("Pi is roughly %f" % (4.0 * count / n))
-                
-                              spark.stop()
-                    - id: hello
-                      type: io.kestra.plugin.spark.SparkCLI
-                      docker:
-                        networkMode: host
-                      commands:
-                        - spark-submit --name Pi --master spark://localhost:7077 pi.py"""
+                            spark.stop()
+                    docker:
```

Comment:
Can you add a image?
![Screenshot 2024-06-11 at 14 08 57](https://github.com/kestra-io/plugin-spark/assets/34094921/1f07c017-41d2-4bc8-b515-71d27aa81b2b)
```suggestion
                    docker:
```